### PR TITLE
feat: updated to version 1.4.0

### PR DESCRIPTION
**Pull request by ghgamble**

### CODE REVIEW

File: src/util/metadata.js [hunk 1]
```diff
@@ -603,6 +603,10 @@ export default {
     keywords: 'help life ring support',
     categories: ['ui'],
   },
+  Lifeascent: {
+    keywords: '',
+    categories: ['ui'],
+  },
```
Comment (ynotdraw) [inline src/util/metadata.js:609]: @dexterca may have thoughts on this as well, but I feel like this should be camel case instead.  If that's what we want, we'd need to rename the following in Chromicons:

```
src/raw/lined/lifeascent.svg
```

to

```
src/raw/lined/life-ascent.svg
```

Doing so will make this `LifeAscent` instead which I think reads a bit easier

Comment (dexterca) [reply src/util/metadata.js:609]: I like `life-ascent.svg`

Comment (ghgamble) [reply src/util/metadata.js:609]: So, I'd do that before merging this PR? Or merge this and then edit the chromicons and update the version?

Comment (ynotdraw) [reply src/util/metadata.js:609]: Doesn't make a huge difference, but I'd probably update chromicons first, then come back and update this PR

File: src/util/metadata.js [hunk 2]
```diff
@@ -915,6 +919,10 @@ export default {
     keywords: '',
     categories: ['ui'],
   },
+  Quote: {
+    keywords: '',
```
Comment (dexterca) [inline src/util/metadata.js:923]: Can you add the keyword `comment` here please?

**Review by ynotdraw — APPROVED** [review]

**Pull request merged**